prymer/api/picking.py

```diff
@@ -30,7 +30,7 @@
 from prymer.model import Oligo
 from prymer.model import PrimerPair
 from prymer.model import Span
-from prymer.primer3 import PrimerAndAmpliconWeights
+from prymer.primer3 import AmpliconParameters
```

Comment:
If we're going to rename it from `PrimerAndAmplicon` I'd vote for `PrimerParameters` as it feels like the majority of the parameters are around primers.  Also, can be used for designing just left primers (no amplicon), just right primers (no amplicon) or both (and therefore an amplicon).